Source/ACE.Server/WorldObjects/WorldObject_Properties.cs

```diff
@@ -329,14 +329,14 @@ public void RemovePosition(PositionType positionType)
         {
             Positions.Remove(positionType);
 
-            if (Biota.TryRemovePosition(positionType, out var entity, biotaPropertiesPositionLock) && ExistsInDatabase && entity.Id != 0)
+            if (Biota.TryRemovePosition(positionType, out var entity, biotaPropertiesPositionLock) && ExistsInDatabase && entity.ObjectId != 0)
                 DatabaseManager.Shard.RemoveEntity(entity, null);
         }
```

Comment:
If we remove the auto generated id and now rely on the biota's id as the primary key for the record, this code will break.

entity.Objectid will always be non-zero as it's generated by ACE to put the object into the world.

entity.Id would be non-zero until it's saved to the db. At that point the db gives it a unique record id.